api/controllers/game/stalemate-reject.js

```diff
@@ -5,24 +5,17 @@ module.exports = async function (req, res) {
   try {
     const game = await Game.findOne({ id: req.session.game }).populate('players');
 
-    const gameUpdates = {
-      turnStalemateWasRequestedByP0: -1,
-      turnStalemateWasRequestedByP1: -1,
-    };
-
     const victory = {
       gameOver: false,
       winner: null,
       conceded: false,
     };
 
-    await Game.updateOne({ id: game.id }).set(gameUpdates);
-
     Game.publish([ game.id ], {
-      change: 'rejectStalemate',
+      change: 'stalemateReject',
       game,
       victory,
-      requestedByPNum: req.session.pNum,
+      playedBy: req.session.pNum,
```

Comment:
Replaced the `requestedByPNum` attribute with `playedBy` for consistency with the GameState API and across moves. Now every move has a `playedBy` instead of different names for the same data (who did the thing) based on which move was made